encryptonize.go

```diff
@@ -430,3 +433,35 @@ func (e *Encryptonize) GetGroupData(authorizer *SealedUser, group *SealedGroup)
 
 	return plainGroup.Data, nil
 }
+
+////////////////////////////////////////////////////////
+//                       Index                        //
+////////////////////////////////////////////////////////
+
+// NewIndex creates an instance of index which is a building block of searchable encryption.
+// An index contains a mapping from labels to sealedIDs. A label is based on a keyword, and
+// a sealedID is based on the occurrence of that keyword in a specific ID (e.g. a document).
+// Hence, index makes it possible to manage which keywords are contained in which IDs.
```

Comment:
I think this description is too focused on the implementation details, which are not really relevant to the caller. What the caller needs to know is that this creates a new encrypted index and that this index can be used to map "keywords" to "IDs" without revealing the mapping.

Reply:
Fixed